GiniVision/Classes/Core/Multipage Review/MultipageReviewPagesCollectionCell.swift

```diff
@@ -220,4 +216,16 @@ final class MultipageReviewPagesCollectionCell: UICollectionViewCell {
             stateView.update(to: .loading)
         }
     }
+    
+    class func size(in collection: UICollectionView) -> CGSize {
+        let collectionInset = UIEdgeInsets(top: 16, left: 0, bottom: 16, right: 0)
+        let height = collection.frame.height -
+            collectionInset.top -
+            collectionInset.bottom +
+            shadowHeight +
+        shadowRadius
+        let width = height * 11 / 20
```

Comment:
`let width = height * aspectRatio`? Wouldn't it be clearer?

Reply:
It is. 👍